src/mruby-widget-lib/src/gem.c

```diff
@@ -818,16 +822,31 @@ remote_cb(const char *msg, void *data)
         remote_cb_int(msg, cb);
     else if(!strcmp("f", arg_str)) {
         float val = rtosc_argument(msg, 0).f;
-		if(cb->log) {
-			if (cb->min > 0){
-				const float b = log(cb->min);
-				const float a = log(cb->max)-b;
-				val = (logf(val)-b)/a; // inverse scaling function of exp-function in mrb_remote_param_set_value
-			}else { // min <= 0
-				const float a = logf(1.0f+4096.0f);
-				val = logf(1.0f+val*4096.0f/cb->max)/a; // inverse function of exp-function in mrb_remote_param_set_value
-			}
-		} else
+        if(cb->log) {
+            // clip values around [min,logmin]
+            if (val > cb->min && val < cb->logmin) {
+                val = (val < .5f * (cb->min + cb->logmin)) ? cb->min : cb->logmin;
```

Comment:
I suspect it will be obvious tomorrow, but why is this clipping needed?

Reply:
My idea was that, if the remote sends any value between `min` and `logmin`, this will be fixed. Without this line, such values in between would be problematic in the subsequent code: They are neither `min`, but they are also not in the log's "safe" range `[logmin, max].`.

Of course, I would hope that every reasonable remote does not send such values, but e.g. if someone changes the logmin value only in zyn, this might happen.

Does this make sense somehow?

Reply:
Yeah, I guess this makes enough sense. The midpoint between logmin and min is arbitrary, but as a workaround for broken messages, this makes sense.